exporter/elasticsearchexporter/internal/serializer/otelserializer/profile.go

```diff
@@ -64,6 +72,34 @@ func SerializeProfile(resource pcommon.Resource, scope pcommon.InstrumentationSc
 				return err
 			}
 		}
+
+		for _, frame := range payload.UnsymbolizedLeafFrames {
+			docID := frame.String()
+			doc := serializeprofiles.LeafFrameSymbolizationData{
```

Comment:
This is not the behavior we're using for other data types, where the type is build in `transform`, and the role of this file is just to encode and export.

How about building the `LeafFrameSymbolizationData` within profile, and pushing the data in here?

Reply:
Done